CONTRIBUTING.rst

```diff
@@ -0,0 +1,230 @@
+affected by this issue - **The issue has been closed**: - **It has been
+closed by a merged pull request** (1) update your dirty_cat version, or
+(2) the fix has not been released in a version yet - **Otherwise**,
+there might be a ``wontfix`` label, and / or a reason at the bottom of
+the conversation - If not - `File a new
+issue <https://github.com/dirty-cat/dirty_cat/issues/new>`__ (see
+following section)
+
+How do I submit a (good) bug report?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To solve your issue as soon as possible, explain the problem and include
+additional details to help maintainers easily reproduce the problem:
+
+-  **Use a clear and descriptive title** which identifies the problem
+-  **Describe the result you expected**
+-  **Add additional details to your description problem** such as
+   situations where the bug should have appeared but didn’t
+-  **Include a snippet of code that reproduces the error**, as it allows
+   maintainers to reproduce it in a matter of seconds!
+-  **Specify versions** of Python, dirty_cat, and other dependencies
+   which might be linked to the issue (e.g., scikit-learn, numpy,
+   pandas, etc.). You can get these versions with ``pip3 freeze``
+   (``pip freeze`` for Windows)
+
+Of course, some of these bullet points might not apply depending on the
+kind of error you’re submitting.
+
+Suggesting enhancements
+~~~~~~~~~~~~~~~~~~~~~~~
+
+This section will guide you through submitting a new enhancement for
+dirty_cat, whether it is a small fix or a new feature.
+
+First, you should `check if the feature has not already been proposed or
+implemented <https://github.com/dirty-cat/dirty_cat/pulls?q=is%3Apr>`__.
+
+If not, the next thing you should do, before writing any code, is to
+`submit a new
+issue <https://github.com/dirty-cat/dirty_cat/issues/new>`__ proposing
+the change.
+
+How do I submit a (good) enhancement proposal?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+-  **Use a clear and descriptive title**
+-  **Provide a quick explanation of the goal of this enhancement**
+-  **Provide a step-by-step description of the suggested enhancement**
+   with as many details as possible
+-  **If it exists elsewhere, link resources**
+
+Of course, some of these bullet points might not apply depending on the
+kind of enhancement you’re submitting.
+
+If the enhancement is validated
+'''''''''''''''''''''''''''''''
+
+Let maintainers know whether : - **You will write the code and submit a
+PR**. Writing the feature yourself is the fastest way to getting it
+implemented in the library, and we’ll help in that process if guidance
+is needed! To go further, refer to the section *Writing your first Pull
+Request*. - **You won’t be able to write the code**, in which case a
+developer interested in the feature can start working on it. Note
+however that maintainers are **volunteers**, and therefore cannot
+guarantee how much time it will take to implement the change.
```

Comment:
These should be bullet points too
```suggestion
Let maintainers know whether : 
- **You will write the code and submit a PR**. 
  Writing the feature yourself is the fastest way to getting it
  implemented in the library, and we’ll help in that process if guidance
  is needed! To go further, refer to the section *Writing your first Pull Request*.
- **You won’t be able to write the code**, in which case a
  developer interested in the feature can start working on it. Note
  however that maintainers are **volunteers**, and therefore cannot
  guarantee how much time it will take to implement the change.
```